.github/workflows/build-and-test-freebsd.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Test (FreeBSD)
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'main'
+      - 'devel/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.event.pull_request.draft == false
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Get number of CPU cores
+      uses: SimenB/github-actions-cpu-cores@v2
+      id: cpu-cores
+
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Checkout Test Files
+      uses: actions/checkout@v4
+      with:
+        repository: neomutt/neomutt-test-files
+        path: test-files
+
+    - name: Build and Test on FreeBSD
+      uses: vmactions/freebsd-vm@v1
+      with:
+        usesh: true
+        prepare: |
+          pkg install -y perl5 pkgconf gettext libxslt docbook-xsl db5 gdbm gpgme gnutls kyotocabinet tokyocabinet lmdb lua54 pcre2 notmuch qdbm rocksdb cyrus-sasl tdb zstd
+        run: |
+          cd test-files
+          ./setup.sh
+          cd -
+          mkdir build
+          cd build
+          ../configure --full-doc --autocrypt --bdb --fmemopen --gdbm --gnutls --gpgme --gss --kyotocabinet --lmdb --lua --lz4 --notmuch --pcre2 --qdbm --rocksdb --sasl --tdb --testing --tokyocabinet --with-lock=fcntl --zlib --zstd
+          make -j ${{steps.cpu-cores.outputs.count}} neomutt
+          ./neomutt -v
+          ./neomutt -h all
+          pwd
+          ls -C
```

Comment:
I was going to ask why these two lines are here, but I've just spotted that you've copy'n'pasted from `build-and-test.yml`.
Would you mind killing my leftover debugging code, [there](https://github.com/neomutt/neomutt/blob/main/.github/workflows/build-and-test.yml) too.
Thanks :-)